spec/models/moment_spec.rb

```diff
@@ -18,24 +18,44 @@
 #
 
 describe Moment do
```

Comment:
TABS!
😱 😱 😱 

Looks like someone had dropped tabs into this file so Atom has assumed that was the desired outcome and formatted all your code this way.